pkg/liqoctl/move/handler.go

```diff
@@ -165,7 +165,7 @@ func getResticRepositoryURL(ctx context.Context, cl client.Client, isLocal bool)
 		}
 	}
 
-	return fmt.Sprintf("rest:http://%s.%s.svc.cluster.local:%d/", resticRegistry, namespace, resticPort), nil
+	return fmt.Sprintf("rest:http://%s.%s.svc:%d/", resticRegistry, namespace, resticPort), nil
```

Comment:
Why is it necessary?

Reply:
Yes, it creates clusters with a different domain to avoid conflicts with the main cluster's DNS resolution